src/components/data-exploration/differential-expression-tool/PathwayAnalysisModal.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import {
+  Modal, Alert, Radio, Space, InputNumber, Select,
+} from 'antd';
+import PropTypes from 'prop-types';
+import AdvancedFilteringModal from './AdvancedFilteringModal';
+
+const PathwayAnalysisModal = (props) => {
+  const { onCancel } = props;
+  const [externalService, setExternalService] = useState('pantherdb');
```

Comment:
Can you move the values of the external service into an object which contain constants?